pandas/tests/indexes/multi/test_format.py

```diff
@@ -57,49 +57,6 @@ def test_repr_with_unicode_data():
         assert "\\u" not in repr(index)  # we don't want unicode-escaped
 
 
-def test_repr_roundtrip():
-
```

Comment:
Note the new implementation breaks round-tripping. This is a worthwhile trade-off as we better clarity with the new repr IMO.

Reply:
can you put back a test that assert that this raises on round-trip now though. (just a simple example is enough with a comment)